app/models/standards_import.rb

```diff
@@ -1,5 +1,6 @@
 class StandardsImport < ApplicationRecord
   has_many_attached :files
+  has_many :imports, dependent: :destroy, as: :parent
```

Comment:
Super minor and picky, but can we put the `as: :parent` after the `has_many :imports` to have consistency with the other files?